src/Symfony/Component/HttpFoundation/InputBag.php

```diff
@@ -76,7 +76,7 @@ public function add(array $inputs = [])
      */
     public function set(string $key, $value)
     {
-        if (!is_scalar($value) && !\is_array($value) && !method_exists($value, '__toString')) {
+        if (null !== $value && !is_scalar($value) && !\is_array($value) && !method_exists($value, '__toString')) {
             trigger_deprecation('symfony/http-foundation', '5.1', 'Passing "%s" as a 2nd Argument to "%s()" is deprecated, pass a string or an array instead.', get_debug_type($value), __METHOD__);
```

Comment:
This message needs updating to include the fact that null is permitted.

Reply:
In that case, the phpdoc should be updated too (`string|array|null`)

Reply:
The phpdoc actually needs updating on an earlier branch, in a separate PR, I think?

Reply:
This feels like a fix to me, so this PR should target the 5.1 branch instead.

Reply:
Yes, but this phpdoc change should go into 3.4.

Reply:
Well, the `InputBag` class doesn't exist in 3.4, it was introduced since 5.1.

Reply:
@yceruto I have updated the phpdoc. Should I retarget this to 5.1 somehow or should you?